tests/sentry/api/serializers/test_grouptagkey.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
You don't really need this here.

Reply:
🤔 Cargo culted from the other tests tbh.